### PR TITLE
Deps/update to Decidim 0.26 (& 0.25 with webpacker)  

### PR DESCRIPTION
**Pull request by agustibr**
- [x] webpack stuff
- [x] styles working
- [x] javascript working
- [x] Mailing
- [x] Overrides checks
- [x] Tests
- [x] API /graphql

### CODE REVIEW

File: app/views/layouts/platoniq/_mini_footer.html.erb [hunk 1]
```diff
@@ -14,23 +14,23 @@
...
+        <%= image_pack_tag("media/images/cc-badge.png", alt: "Creative Commons License" ) %>
       </a>
       <%= t("platoniq.layouts.footer.made_by_platoniq").html_safe %>
     </div>
     <div class="medium-6 large-4 column">
       <% if current_organization.official_img_footer? %>
         <div class="decidim-logo">
         <%= link_to  current_organization.official_url do %>
-          <%= image_tag current_organization.official_img_footer.url.to_s , alt: current_organization.name %>
+          <%= image_pack_tag current_organization.official_img_footer.url.to_s , alt: current_organization.name %>
```
Comment (microstudi) [inline app/views/layouts/platoniq/_mini_footer.html.erb:25]:
```suggestion
<% if current_organization.official_img_header.attached? %>
        <%= image_tag current_organization.attached_uploader(:official_img_header).path, class: "minimap-official_img_header" %>
      <% else %>
        <span class="float-right"><%= t("official_img_header", scope: "activemodel.attributes.organization") %> &gt; </span>
        <%= content_tag :div, nil, { class: "minimap-official_img_header placeholder float-right", style: "width: #{image_width(current_organization, :official_img_header) * size_factor }px; height: #{image_height(current_organization, :official_img_header) * size_factor }px;" } %>
      <% end %>
```

image_pack_tag can only be used for static assets. Check this example https://github.com/decidim/decidim/blob/c9ed6740e8025fd482b49dad71cc1788b5e572c0/decidim-admin/app/views/decidim/admin/organization_appearance/form/_minimap.html.erb

File: config/assets.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+# Define the base path of your module. Please note that `Rails.root` may not be
+# used because we are not inside the Rails environment when this file is loaded.
+# base_path = File.expand_path("..", __dir__)
+
+# If you want to import some extra SCSS files in the Decidim main SCSS file
+# without adding any extra stylesheet inclusion tags, you can use the following
+# method to register the stylesheet import for the main application. This would
+# include an SCSS file at `app/packs/stylesheets/your_app_extensions.scss` into
+# the Decidim's main SCSS file.
+Decidim::Webpacker.register_stylesheet_import("stylesheets/coincidim")
```
Comment (microstudi) [inline config/assets.rb:14]: There's an alternative to this (and this file can be empty).
Require the endpoints in the default file for a decidim application, like don in here:
https://github.com/Platoniq/decidim-indices/blob/main/app/packs/stylesheets/decidim/decidim_application.scss

File: config/initializers/decidim.rb [hunk 3]
```diff
@@ -125,3 +125,6 @@
 
 Rails.application.config.i18n.available_locales = Decidim.available_locales
 Rails.application.config.i18n.default_locale = Decidim.default_locale
+
```
Comment (microstudi) [inline config/initializers/decidim.rb:128]:
```suggestion
Rails.application.config.to_prepare do
  # Api tunnings
  Decidim::Api::Schema.max_complexity = 5000
  Decidim::Api::Schema.max_depth = 50
end
```
Can you add this? nothing related to the update but needed to make the awesome map work properly

**Review by microstudi — CHANGES_REQUESTED** [review]
Thanks @agustibr 
I think there's only one thing that needs to be changed related to the use of image_pack_tag.

As per the assets definition, there's an alternative approach that can be used if you prefer it (done it in https://github.com/Platoniq/decidim-indices/tree/main/app/packs)

**Comment by agustibr** [comment]
Thanks @microstudi for the review, I'll push the suggested changes and ping you back later 😄

**Review by microstudi — CHANGES_REQUESTED** [review]
ok, everything looks good.

There's only one thing missing, email customization.

Emails look like this in the original coincidim:
image

Indices is also customized, so we should review how it's done there:
image

**Comment by agustibr** [comment]
Thanks for the heads up @microstudi 😁  

The email customization is fixed now!

This is how the email is rendered _locally_:

Screenshot from 2022-03-18 09-00-37

**Review by microstudi — APPROVED** [review]
Perfec @agustibr 
thanks on this one!

**Pull request merged**